include/net/ieee802154_radio.h

```diff
@@ -109,6 +121,7 @@ struct ieee802154_config {
 		/** ``IEEE802154_CONFIG_AUTO_ACK_FPB`` */
 		struct {
 			bool enabled;
+			enum ieee802154_fpb_mode mode;
```

Comment:
Now I wonder, if we should just add `IEEE802154_FPB_ADDR_MATCH_DISABLED` and get rid of the bool.